python/lsst/afw/cameraGeom/cameraFactory.py

```diff
@@ -87,7 +87,7 @@ def makeCameraFromPath(cameraConfig, ampInfoPath, shortNameFunc):
     @return camera (an lsst.afw.cameraGeom.Camera)
     """
     ampInfoCatDict = dict()
-    for detectorConfig in cameraConfig.detectorList.itervalues():
+    for detectorConfig in cameraConfig.detectorList.values():
```

Comment:
Why is something called {{dectectorList}} a dict? And don't we need {{from builtins import dict}} here as well to make it efficient?